src/lib.rs

```diff
@@ -226,32 +226,43 @@ impl Texture {
     }
 }
 
+/// The supported output color spaces.
+pub enum ColorSpace {
+    Linear,
+    Srgb,
+}
+
 /// Configuration for the renderer.
-pub struct RendererConfig<'vs, 'fs> {
+pub struct RendererConfig<'s> {
     pub texture_format: TextureFormat,
     pub depth_format: Option<TextureFormat>,
     pub sample_count: u32,
-    pub vertex_shader: Option<ShaderModuleDescriptor<'vs>>,
-    pub fragment_shader: Option<ShaderModuleDescriptor<'fs>>,
+    pub shader: Option<ShaderModuleDescriptor<'s>>,
+    pub color_space: ColorSpace,
```

Comment:
We should ask for a vertex_shader_entry_point and fragment_shader_entry_point that way we don't force the idea of caring about spaces onto the user.

Reply:
You're right, that seems more general. See the latest commit.